MetaMask/metamask-extension
# Ux: Icon: Improve alignment of asset list chevron

## Explanation

The asset list's send icon, in fullscreen, is slightly off-centered vertically when the asset list item is hovered.  This PR fix that issue.

## Screenshots/Screencaps

<img width="912" alt="Chev" src="https://user-images.githubusercontent.com/46655/219538405-04d9043a-3b68-4bc5-8b3b-5cb68dedbf7f.png">


## Manual Testing Steps

1.  Add Chainlink Token to MetaMask
2. Hover over Chainlink on the home screen
3. See vertically centered text and chevron

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
